### DISCUSSION TITLE
有更平滑的方法过渡到新版本吗?

### DISCUSSION THREAD

**Discussion by yangliguo7** [element-plus/element-plus#4635] [Q&A] [answered]
我们之前一直是基于低版本开发的项目，但是现在迫不得已、我们必须跟新这个依赖，不幸的是当我们跟新到最新的依赖后，并不支持以前的写法，例如icon等，所以有更平滑的方法不需要我本地改动代码就能使用的吗

**Comment by YunYouJun** [comment, marked as answer]
您可以尝试找到更新图标之前的最小版本。

element-plus 的迭代确实十分激进，这也是阵痛。对此我们很抱歉。
不过关于图标方面，建议您完全可以尝试一下。SVG 的图标方案也会是未来的主流方案。
